cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -71,10 +70,10 @@ message Partition {
 }
 
 /**
-Specifies the cutout "shape"
+Specifies the volume "shape"
 **/
-message CutoutDef {
-    string json = 1;  // Uses the Cutout Definition schema
+message VolumeDef {
+    string json = 1;  // Uses the Volume Definition schema
```

Comment:
TBH I wish we could model this in Protobuf rather than just using an inline JSON blob...